src/app/catalogue-search/catalogue-search-listing/catalogue-search-listing.component.ts

```diff
@@ -91,6 +91,7 @@ export class CatalogueSearchListingComponent implements OnInit {
    * Update the search by using the state router.
    */
   updateSearch() {
+    this.status = 'loading';
```

Comment:
I think this line was added back during a rebase, but it shouldn't be here. Please could this be removed as it fixes bug #570?